### PR TITLE
JNG-4471 Release judo generator commons

### PR DESCRIPTION
**Pull request by gaborflorian**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-4471" title="JNG-4471" target="_blank">Sub-taskJNG-4471</a>  Release judo-generator-commons
  </td></table>
  <br />

### CODE REVIEW

**Review by bencegelei — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -141,7 +141,7 @@ jobs:
               NEXT_VERSION="${NEXT_VERSION}-SNAPSHOT"
           fi
           echo "Next version is: $NEXT_VERSION"
-          echo "::set-output name=version::${NEXT_VERSION}"
+          echo "VERSION=${NEXT_VERSION}" >> $GITHUB_OUTPUT
 
       - name: Set next version
         run: |-
```
Comment (robertcsakany) [inline .github/workflows/release.yml]: Is the reference matching with the output name?

File: .github/workflows/create-release-tagged.yml [hunk 2]
```diff
@@ -35,8 +35,8 @@ jobs:
           version=$(echo ${{ github.ref }} | cut -d/ -f4)
           branch=$(echo ${{ github.event.base_ref }} | cut -d/ -f3)
           echo "Version: $version"
-          echo "::set-output name=version::$version"
-          echo "::set-output name=branch::$branch"
```
Comment (robertcsakany) [inline .github/workflows/create-release-tagged.yml:39]: User lowercase characters:

 echo "version=$version" >> $GITHUB_OUTPUT
 echo "branch=$branch" >> $GITHUB_OUTPUT

**Review by robertcsakany — CHANGES_REQUESTED** [review]

File: .github/workflows/create-release-tagged.yml (hunk 2 above)
Comment (gaborflorian) [reply .github/workflows/create-release-tagged.yml:39]: [fd3af97](https://github.com/BlackBeltTechnology/judo-generator-commons/pull/19/commits/fd3af97cbc5d247c8325d245b1dd3b67306bf939)

File: .github/workflows/release.yml (hunk 1 above)
Comment (gaborflorian) [reply .github/workflows/release.yml]: [fd3af97](https://github.com/BlackBeltTechnology/judo-generator-commons/pull/19/commits/fd3af97cbc5d247c8325d245b1dd3b67306bf939)

Comment (gaborflorian) [reply .github/workflows/release.yml]: Yes it is matching.

for example:
in  Calculate version number
echo "VERSION=${VERSION_NUMBER}" >> $GITHUB_OUTPUT 
used in
Create version tag 
steps.version.outputs.version is the calculated version and the created version tag has correct version.

But for the consistency, I renamed the set-output variable names with lowercase characters.

**Review by robertcsakany — APPROVED** [review]

**Review by fmagnucz — APPROVED** [review]

**Pull request merged**